src/components/Radio/radio.module.css

```diff
@@ -0,0 +1,50 @@
+.colorRadio,
+.sizeRadio {
+  display: none;
+}
+
+.title {
+  font-family: Poppins;
+  font-size: var(--font-size-md);
+  font-weight: var(--font-weight-bold);
+}
+
+.listColor {
+  display: flex;
+  gap: 0.625rem;
+  margin-top: 1rem;
+}
+
+.colorCircle {
+  width: 1rem;
+  height: 1rem;
+  border-radius: 50%;
+  border: var(--border-sm) solid var(--color-text-product);
+}
+
+.colorRadio:checked + .colorOption > .colorCircle {
+  background-image: url('../../assets/images/checked.svg');
```

Comment:
The checked base on color select, not all by blue color.

Reply:
Fixed